app/src/main/java/com/example/househomey/AddItemFragment.java

```diff
@@ -95,19 +111,28 @@ private void addItem() {
         data.put("serialNumber", getInputText(R.id.add_item_serial_number));
         data.put("comment", getInputText(R.id.add_item_comment));
 
+        // Ensure that form data can be used to create a valid Item
+        Item newItem;
```

Comment:
This is out of curiosity more than anything. Does Java have a problem with this being initialized in the try catch? With the return in the catch we shouldn't need to split these up I would think?

Reply:
Yep the IDE complained about it. Annoying